solutions/4-getting-started-js/contains.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line no-var
+var contains = function (element, arr) {
+  return arr.find(item => item === element) !== undefined
```

Comment:
It's a better idea not to do any checks with undefined. Better return !! the result of .find().